src/components/CohortBuilder/Summary/AgeDiagChart.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import { compose } from 'recompose';
+import { withTheme } from 'emotion-theming';
+import { get, inRange } from 'lodash';
+import VerticalBar from 'chartkit/components/VerticalBar';
+
+const ageAtDiagnosisTooltip = data => {
+  return `${data.value.toLocaleString()} Participant${data.value > 1 ? 's' : ''}`;
+};
+
+const AgeDiagChart = ({ data, theme }) => (
+  <VerticalBar
+    data={data}
+    indexBy="label"
+    tooltipFormatter={ageAtDiagnosisTooltip}
+    sortByValue={true}
+    height={225}
+    colors={[theme.chartColors.lightblue]}
+  />
+);
+
+export const ageDiagQuery = sqon => ({
+  query: `query($sqon:JSON) {
+    participant {
+      aggregations(filters: $sqon) {
+        diagnoses__age_at_event_days {
+          histogram(interval: 1){
```

Comment:
I think an interval of `365` and further bucketing based on years would relieve a lot of CPU cycles in the client :)

Reply:
@hlminh2000 good shout!

Reply:
@hlminh2000 364? 0 - 364 for newborn etc

Reply:
Depends on the year ;)

Reply:
Might be of interest, or not:
https://pumas.jpl.nasa.gov/files/04_21_97_1.pdf